cli/chaosli/cmd/create.go

```diff
@@ -599,14 +599,14 @@ func getPulse() *v1beta1.DisruptionPulse {
 
 	activeDuration := v1beta1.DisruptionDuration(getInput(
 		"What would be the duration of the disruption in an active state during the pulse? This can be a golang's time.Duration.",
-		"Please specify a golang's time.Duration's >500ms, e.g., \"45s\", \"15m30s\", \"4h30m\".",
+		fmt.Sprintf("Please specify a golang's time.Duration's >%dms, e.g., \"45s\", \"15m30s\", \"4h30m\".", types.PulsingDisruptionMinimumDuration.Milliseconds()),
```

Comment:
Rather then putting `duration.Milliseconds()` into `%dms`, you can just put `duration` into `%s`, as the duration already includes its type (or it should)